frontend/src/hooks/useChangeRequestsEnabled.ts

```diff
@@ -1,10 +1,39 @@
+import React from 'react';
 import useUiConfig from './api/getters/useUiConfig/useUiConfig';
+import { useChangeRequestConfig } from './api/getters/useChangeRequestConfig/useChangeRequestConfig';
 
-export const useChangeRequestsEnabled = (environment?: string) => {
-    // it can be swapped with proper settings instead of feature flag
+export const useChangeRequestsEnabled = (projectId: string) => {
     const { uiConfig } = useUiConfig();
-    return (
-        Boolean(uiConfig?.flags?.changeRequests) &&
-        (environment === 'production' || typeof environment === 'undefined')
+    const { data } = useChangeRequestConfig(projectId);
+
+    const isChangeRequestEnabled = React.useCallback((): boolean => {
```

Comment:
isChangeRequestFlagEnabled to differentiate from changeRequestEnabled in config?